styles.css

```diff
@@ -0,0 +1,586 @@
+}
+
+.datediv {
+  width: 100%;
+}
+
+.dateheading {
+  color: #272a31;
+  font-weight: 800;
+}
+
+.locationpara {
+  color: #272a31;
+}
+
+.headlinedesktop {
+  display: none;
+}
+
+#main-section {
+  text-align: center;
+  width: 100%;
+}
+
+.cover-program {
+  background-image: url('./images/patternbg.png');
+  background-repeat: repeat;
+  background-position: center;
+  background-color: rgb(39, 42, 50);
+  padding: 5px;
+  width: 100%;
+  height: 750px;
+}
+
+.hr {
+  width: 30px;
+  border-color: #ec5242;
+  margin-bottom: 16px;
+}
+
+.program-list h2 {
+  color: #fff;
+  margin-top: 20px;
+}
+
+.bg-program {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  height: 100px;
+  margin-top: 10px;
+  background: rgba(255, 255, 255, 0.1);
+  width: 99%;
+}
+
+.bg-program img {
+  padding: 0%;
+}
+
+.bg-program li h3 {
+  color: #ec5242;
+  font-size: 14px;
+  text-align: left;
+  padding-left: 2px;
+  margin-left: 2px;
+  width: 99%;
+}
+
+.bg-program li p {
+  font-size: 12px;
+  color: #fff;
+  margin-left: 2px;
+  padding: 2px;
+}
+
+.join-btn {
+  width: auto;
+  background-color: #ec5242;
+  margin-top: 4rem;
+  position: relative;
+  -ms-transform: translateY(-50%);
+  transform: translateY(-50%);
+  padding: 1.5rem 2rem;
+  border: none;
+  align-items: center;
+  text-align: center;
+}
+
+.buttonlink {
+  text-decoration: none;
+  color: #fff;
+  font-size: 24px;
+}
+
+.cover-program-desktop {
+  display: none;
+}
+
+.hr1 {
+  width: 30px;
+  border-color: #ec5242;
+  margin-bottom: 16px;
+}
+
+.featuredh {
+  text-align: center;
+  color: #272a31;
+  font-weight: 600;
+  font-size: 22px;
+  margin-top: 35px;
+}
+
+.image-of-background {
+  background-image: url('./images/speaker_bg.jpg');
+  width: 100px;
+  height: 100px;
+  margin-left: 10px;
+  float: left;
+}
+
+.featured-speakers {
+  position: relative;
+  top: 14px;
+  left: 14px;
+  width: 100px;
+  height: 100px;
+}
+
+.fetured-text {
+  float: left;
+  margin-left: 10px;
+  width: 70%;
+  overflow: visible;
+}
+
+.featured-heading {
+  margin-left: 15px;
+  color: #272a31;
+}
+
+.featured-para {
+  margin-left: 10px;
+  font-size: 12px;
+  color: #ec5242;
+}
+
+.featured-speaker::after {
+  clear: both;
+  content: "";
+  display: table;
+}
+
+.featured-span {
+  margin-left: 10px;
+  font-size: 12px;
+  color: #272a31;
+}
+
+.see-more {
+  border: 1px solid #272a31;
+  border-radius: 5px;
+  padding: 15px 20px;
+  width: 90%;
+  margin-top: 35px;
+  margin-bottom: 20px;
+  position: relative;
+  left: 20px;
+}
+
+#featured-desktop {
+  display: none;
+}
+
+.partner-section {
+  background-color: rgb(39, 42, 50);
+  height: 550px;
+}
+
+.logosection-heading {
+  text-align: center;
+  color: #272a31;
+  font-weight: 600;
+  font-size: 35px;
+  margin-top: 35px;
+}
+
+.partner-heading {
+  color: #fff;
+}
+
+.partner-div {
+  text-align: center;
+}
+
+.companies {
+  width: 150px;
+  height: 150px;
+}
+
+.footer-div {
+  text-align: center;
+}
```

Comment:
- [ ] Kindly make sure to align the footer content correctly in order to match the provided design as much as possible on the desktop version.

- [ ] KIndly make sure to use the same background color of the footer on the `About` page in order to match the provided design.

   - ### YOUR DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/219402840-aa4ab845-f10a-4aaf-abf7-587ad1027eef.png)

   - ### PROVIDED DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/219403033-9d94ec79-b45d-4dee-b917-1df4cd6b4448.png)

  - ### ABOUT PAGE FOOTER
    ![image](https://user-images.githubusercontent.com/87197412/219404210-8e56af9d-9695-436d-86a5-d05ffafb7bb3.png)

Reply:
I have incorporated the suggested changes. Screen shot attached below for the Home page.
![image](https://user-images.githubusercontent.com/117091923/219475316-c1b1bd99-4c05-491c-885b-47b1cdafd374.png)

Reply:
Suggested changes incorporated for the about page using dark background. Screen shot attached below for your kind reference.
![image](https://user-images.githubusercontent.com/117091923/219481285-387aa1bc-bb27-4b5f-8123-dcc15f583259.png)